src/encoder/encoderfdkaac.cpp

```diff
@@ -267,27 +268,48 @@ int EncoderFdkAac::initEncoder(int samplerate, QString& errorMessage) {
     m_pAacDataBuffer = new unsigned char[kOutBufferBits * m_channels]();
 
     // AAC Object Type: specifies "mode": AAC-LC, HE-AAC, HE-AACv2, DAB AAC, etc...
-    aacEncoder_SetParam(m_aacEnc, AACENC_AOT, m_aacAot);
+    if (aacEncoder_SetParam(m_aacEnc, AACENC_AOT, m_aacAot) != AACENC_OK) {
+        kLogger.warning() << "aac encoder setting AOT failed!";
```

Comment:
This is an insufficient error message. Store the return value in a variable and print it in the log.

Reply:
This never fails, I am working at an alternative detection method.